utils_cv/detection/plot.py

```diff
@@ -26,9 +27,12 @@ def __init__(
         rect_th: int = 4,
         rect_color: Tuple[int, int, int] = (255, 0, 0),
         text_size: int = 25,
-        text_font: str = "arial.ttf",
+        text_font: str = "DejaVuSerifCondensed.ttf",
         text_color: Tuple[int, int, int] = (255, 255, 255),
     ):
+        if is_windows() == True:
```

Comment:
Does arial.tff not work on Linux? If it does not, how about we find a font which works on all OS rather than the if clause here?

Reply:
i did a (very) quick search to see if there was one, but didnt come across one. i'll double check

Reply:
yeah can't seem to find.. but this post suggets that you have to deal with it separately https://stackoverflow.com/questions/24085996/how-i-can-load-a-font-file-with-pil-imagefont-truetype-without-specifying-the-ab